lib/Controller/ChatController.php

```diff
@@ -818,6 +818,9 @@ public function getMessageContext(
 			$commentsHistory = $this->chatManager->getHistory($this->room, $messageId, $limit, true, $threadId);
 			$commentsHistory = array_reverse($commentsHistory);
 			$commentsFuture = $this->chatManager->waitForNewMessages($this->room, $messageId, $limit, 0, $currentUser, false);
+			$commentsFuture = array_filter($commentsFuture, function (IComment $comment) use ($threadId) {
```

Comment:
If we do this (and I'm not sure about it), it should be in the DB query

Reply:
Sure. That's a temporary workaround